maintenance/wikia/cronjobs/sendWeeklyDigest.php

```diff
@@ -30,13 +32,27 @@ public function __construct() {
 	public function execute() {
 		$this->logRunTime();
 		$watchlistBot = new GlobalWatchlistBot();
-		$watchlistBot->sendWeeklyDigest();
+		try {
+			$watchlistBot->sendWeeklyDigest();
+		} catch ( Exception $e ) {
+			$this->logError( $e );
+		}
 	}
 
 	private function logRunTime() {
 		$message = "sendWeeklyDigest script run at " . date( "F j, Y, g:i a" ) . "\n";
 		$this->output( $message );
 	}
+
+	/**
+	 * If there's a problem when sending the weekly digest (ie, timing out), log the error. We have
+	 * a Kibana check which will send out an alert if any "Weekly Digest Error" messages are sent.
+	 */
+	private function logError( Exception $exception ) {
+		WikiaLogger::instance()->error( 'Weekly Digest Error', [
```

Comment:
FYI, I just added this as the first query on this Kibana board:

https://kibana.wikia-inc.com/index.html#/dashboard/elasticsearch/Error%20Investigation

I feel like we add a lot of these logging statements and I'm not always sure we follow up.